CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   batch update.
 - Added caching for `visibilityMetadata` calculations.
 - Fixed an issue that could cause SwiftUI views to be incorrectly sized in a collection view.
+- Fixed an issue where images were requesting a smaller image than needed due to SwiftUI views being incorrectly sized in a collection view
```

Comment:
I think this CHANGELOG entry is a bit too specific - the more general fix it probably something like:

```suggestion
- Fixed an issue that could cause collection view cells to layout with unexpected dimensions
```